projects/hslayers/src/components/add-data/vector/vector-layer-options.type.ts

```diff
@@ -8,11 +8,15 @@ export type HsVectorLayerOptions = {
   visible?: boolean;
   path?: string;
   fromComposition?: boolean;
+  geomAttribute?: string;
+  idAttribute?: string;
   style?: any;
+  endpointUrl?: string;
```

Comment:
Do you think it would be possible to reduce the duplication between SourceOptions and LayerOptions ? I see query, endpoint and the attributes present in both.

Reply:
Good point. I have to investigate that. I mostly mindlessly filled-in all types and interfaces where I got TS compiler errors.